sdk/python/tests/integration/offline_store/test_universal_historical_retrieval.py

```diff
@@ -37,14 +37,21 @@ def find_asof_record(
     ts_key: str,
     ts_start: datetime,
     ts_end: datetime,
-    filter_key: str = "",
-    filter_value: Any = None,
+    filter_keys: List[str] = [],
+    filter_values: List[Any] = [],
```

Comment:
Oh, Don't have mutable default arguments, this is a very subtle footgun which will almost certainly break:

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

You probably want to do 
```
    filter_keys: Optional[List[str]] = None,
    filter_values: Optional[List[Any]] = None):
        filter_keys = filter_keys or []
        filter_values = filter_values or []
```